data_steward/cdr_cleaner/cleaning_rules/fill_source_value_text_fields.py

```diff
@@ -146,22 +146,30 @@ def get_fields_dict(table_name, fields):
     return fields_to_replace
 
 
-def get_modified_columns(fields, fields_to_replace):
+def get_modified_columns(fields, fields_to_replace, table=''):
```

Comment:
nit: can we use `None` instead an empty string, not sure if this would work though. 
```suggestion
def get_modified_columns(fields, fields_to_replace, table=None):
```

Reply:
Yes we can